manifests/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: objectstorage-controller
```

Comment:
Objects can get prefixed through a `kustomization` if desired, so no real need to do so here. Plain `controller` should do.